web/src/main/java/ua/com/vertex/context/SecurityWebConfig.java

```diff
@@ -97,6 +94,29 @@ protected void configure(HttpSecurity http) throws Exception {
         ;
     }
 
+    private void handleFailure(HttpServletRequest request, HttpServletResponse response, Exception e) throws IOException {
+        String username = request.getParameter("username");
+
+        if (e.getMessage().equals(LOGIN_ATTEMPTS)) {
+            response.sendRedirect(getLink(username));
+
+        } else if (defender.setCounter(request.getParameter("username")) >= maxAttempts) {
+            response.sendRedirect(getLink(username));
+
+        } else if (e instanceof BadCredentialsException) {
+            logger.debug(() -> e);
+            response.sendRedirect("/logIn?error");
+
+        } else {
+            logger.error(() -> e);
+            response.sendRedirect("/error?reason=" + UNKNOWN);
+        }
+    }
+
+    private String getLink(String username) {
+        return String.format("/error?reason=%s&username=%s", ATTEMPTS, username);
```

Comment:
What do you mean like 'transfer'? This constant is used here to create a link gets analyzed in AppErrorController.
User gets redirected to error page because he is already banned, not in order to be banned.

Reply:
Corrected